pom.xml

```diff
@@ -345,35 +345,35 @@
 							<bit>32</bit>
 							<version>${selenium.chromedriver.version}</version>
 							<url>
-								http://chromedriver.storage.googleapis.com/${selenium.chromedriver.version}/chromedriver_win32.zip</url>
-							<fileMatchInside>chromedriver.exe</fileMatchInside>
+								https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/${selenium.chromedriver.version}/win32/chromedriver-win32.zip</url>
```

Comment:
Is this an official download for chomedrivers?

Reply:
Never mind, I found https://googlechromelabs.github.io/chrome-for-testing/#stable